jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -489,6 +489,29 @@ public JibContainerBuilder setCreationTime(Instant creationTime) {
     return this;
   }
 
+  /**
+   * Adds a platform to the platforms configurations for the container.
+   *
+   * @param os the os value
+   * @param architecture the architecture value
+   * @return this
+   */
+  public JibContainerBuilder addPlatform(String os, String architecture) {
+    containerBuildPlanBuilder.addPlatform(os, architecture);
+    return this;
+  }
+
+  /**
+   * Sets the platforms configurations for the container.
+   *
+   * @param platforms the list of platform objects
+   * @return this
+   */
+  public JibContainerBuilder setPlatforms(Set<Platform> platforms) {
+    containerBuildPlanBuilder.setPlatforms(platforms);
+    return this;
+  }
+
```

Comment:
Do not add new methods to `JibContainerBuidler` in this PR. You'll do it in #2589.

Reply:
Done